### PR TITLE
Add contributors section to readme

### PR DESCRIPTION
**Pull request by cbourjau**
Quite overdue! Are you ok with this?

### CODE REVIEW

**Comment by barskern** [comment]
Thank you, I appriciate it! I don't want to be asking to much, but perhaps I could also be added as an `author` in the `Cargo.toml`? :smile:

**Review by barskern — APPROVED** [review]
Thank you again! :+1:

File: README.md [hunk 1]
```diff
@@ -61,3 +61,7 @@ the binary after compilation.
 Future of this crate
 --------------------
 There are currently [open issues](https://github.com/rust-lang/cargo/issues/3670) upstream in cargo which might make this crate redundant in the future.
+
+Contributors
+------------
+This crate would be not be what it is today without the many contributions of [@barskern](https://github.com/barskern)!
```
Comment (barskern) [inline README.md]: There is a typo btw; "would **be** not be what".

**Review by barskern — CHANGES_REQUESTED** [review]
A tiny typo snuck in. :stuck_out_tongue_winking_eye:

File: README.md (hunk 1 above)
Comment (cbourjau) [reply README.md]: Arrrrg! Fixed! Also, I guess its "contributions by..."

**Review by barskern — APPROVED** [review]

**Comment by barskern** [comment]
Thank you very much ! I'm happy to help. :smile:

**Pull request merged**